Makefile

```diff
@@ -52,24 +48,27 @@ docs-serve: ### Serve local docs
 gen-metastore: ## Run Metastore Code generation
 	@thrift -r --gen go --gen go:package_prefix=github.com/treeverse/lakefs/metastore/hive/gen-go/ -o metastore/hive metastore/hive/hive_metastore.thrift
 
-gen-api: ## Run the go-swagger code generator (Docker required)
+gen-api: ## Run the go-swagger code generator
 	@rm -rf $(API_BUILD_DIR)
 	@mkdir -p $(API_BUILD_DIR)
-	$(SWAGGER) generate client -q -A lakefs -f ./swagger.yml -P models.User -t $(API_BUILD_DIR)
-	$(SWAGGER) generate server -q -A lakefs -f ./swagger.yml -P models.User -t $(API_BUILD_DIR) --exclude-main
+	@go get github.com/go-swagger/go-swagger/cmd/swagger
```

Comment:
can't this break when `$GOPATH/bin` is not in `$PATH`?

Reply:
you will get an error
`make: go: No such file or directory`

Reply:
evaluated gobinpath and use it - it will not require you have it in your path